lesson_06/script.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+const ul = document.getElementById("js-list");
+const data = [
+  { to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク" },
+  { to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ" }
+];
+
+const data = new Promise((resolve) => {
```

Comment:
must
This variable name is already in use. So let's make it a different name.

My job here is to get data, so how about `getData`?

Reply:
We have corrected the two points you pointed out.
Thank you for getData　[`fea0c8c`](https://github.com/Numabeharuka/Lesson-js/pull/7/commits/fea0c8ce839c9ae63878918e10f3d7511a59a1a7)